client/types.go

```diff
@@ -111,6 +111,7 @@ type Plan struct {
 	State              string   `xml:"state"`
 	ServiceID          string   `xml:"service_id"`
 	EndUserRequired    string   `xml:"end_user_required"`
+	ApprovalRequired   string   `xml:"approval_required"`
```

Comment:
Changes in `types.go` seem to be part of another PR

Reply:
this is built on top of the latest